lib/model/rwfolder.go

```diff
@@ -1345,7 +1291,12 @@ func (f *sendReceiveFolder) copierRoutine(in <-chan copyBlocksState, pullChan ch
 				state.copyDone(block)
 			}
 		}
-		weakHashFinder.Close()
```

Comment:
Does the `weakHashFinder` no longer require closing? Can we drop in the file close with a defer at the point we do the open instead of in the nil check here?